src/index.js

```diff
@@ -4,7 +4,7 @@
 // used to route based on the request properties (such as method or path), send
 // the request to a backend, make completely new requests, and/or generate
 // synthetic responses.
-addEventListener('fetch', async function handleRequest(event) {
+addEventListener('fetch', function handleRequest(event) {
```

Comment:
I wonder if it might be better to change this in a way that makes it always work, even once you start doing async stuff in the handler. A straightforward way to do that is to have the handler be a simple wrapper that directly calls `event.respondWith`, with the result of calling another function—containing the meat of things—passed in. I.e., something like this:

```suggestion
addEventListener('fetch', event => event.respondWith(handleRequest(event)));

async function handleRequest(event) {
```

And then all uses of `event.respondWith(response)` below changed to instead `return response` (with following returns removed.)

Reply:
I was thinking of shortening the arrow function like this as well. But I was like "eh, it's fine" 😂 

Totally, let's do it this way, I went ahead and committed your suggestion! Thank you! 😄 🎉